python_modules/dagster/dagster/_core/secrets/env_file.py

```diff
@@ -0,0 +1,57 @@
+import os
+from typing import Dict, List, Optional, Tuple
+
+from dagster._config import StringSource
+from dagster._core.utils import parse_env_var
+from dagster._serdes import ConfigurableClass
+from dagster._utils import merge_dicts
+
+from .loader import SecretsLoader
+
+
+class EnvFileSecretsLoader(SecretsLoader, ConfigurableClass):
```

Comment:
[1] i get bad vibes seeing "secrets" name for thing that loads plain text values from file.